recipe/meta.yaml

```diff
@@ -1,17 +1,20 @@
 {% set version = "0.6" %}
-{% set sha256 = "b4be2b3438971588dc306116658f475916755ad8fcc0618e79023846ce94f97b" %}
 
 package:
   name: python-graphviz
   version: {{ version }}
 
 source:
-  fn: python-graphviz-{{ version }}.zip
-  url: https://pypi.io/packages/source/g/graphviz/graphviz-{{ version }}.zip
-  sha256: {{ sha256 }}
+  fn: python-graphviz-{{ version }}.tar.gz
+  url: https://github.com/xflr6/graphviz/archive/{{ version }}.tar.gz
```

Comment:
Do you mean changed checksums from the repo owner redefining the corresponding tag or file corruption from GitHub?

Agreed that PyPI would be nicer: I just couldn't get the `patch` util used to build here work with the files in the PyPI archive, which have `CRLF` line endings (the ones on GitHub have `LF`). Maybe you know this build system better (let it use the `-binary` flag) and have an idea?